### PR TITLE
Feat: Added Homepage with categories

### PR DESCRIPTION
**Pull request by shuntzhere**
Added categories in the database.
Rendered those on the Homepage.

### CODE REVIEW

File: src/components/Category/Category.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.category__name {
+  width: 50%;
+  background-color: var(--color-secondary);
+  margin: 1.5rem 0;
+  padding: 1.5rem;
+  cursor: pointer;
+  box-shadow: 0 5px 60px rgba(0, 0, 0, 0.12), 0 1px 40px rgba(0, 0, 0, 0.24);
```
Comment (rahulcs754) [inline src/components/Category/Category.css:7]: use color variable

File: src/components/Hero/Hero.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.main {
+  width: 1280px;
```
Comment (rahulcs754) [inline src/components/Hero/Hero.css]: try to give min or max width in rem that will responsive

File: src/App.css [hunk 3]
```diff
@@ -1,4 +1,31 @@
-* {
-  margin: 0;
-  padding: 0;
+@import url("https://fonts.googleapis.com/css2?family=Cinzel:wght@400;500;700;900&family=Lobster&family=Rubik:wght@300;400;500;700&display=swap");
```
Comment (coldpigli) [inline src/App.css:1]: import in the index.html file. CSS imports are way slower

**Review by rahulcs754 — APPROVED** [review]
In your code, I am giving some suggestions. 
Check

File: src/components/Category/Category.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Category.css";
+
+export const Category = ({ title, color }) => {
+  return (
+    <div
+      className="category__name endangered"
+      style={{ backgroundColor: color }}
+    >
+      <a href="#" className="category__name--title">
```
Comment (coldpigli) [inline src/components/Category/Category.jsx:10]: Replace anchor tags with Link tags

File: src/components/Hero/Hero.css (hunk 2 above)
Comment (coldpigli) [inline src/components/Hero/Hero.css]: bad practice. set in terms of %. Will create major issues in screen with different pixel density

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import ShoppingCartIcon from "@mui/icons-material/ShoppingCart";
+import PersonIcon from "@mui/icons-material/Person";
+
+export const Navbar = () => {
+  return (
+    <header className="header">
+      <nav className="header__nav flex flex-items-center justify-between">
+        <div className="header__nav--logo">
+          <a className="header__nav--brand" href="#">
```
Comment (coldpigli) [inline src/components/Navbar/Navbar.jsx]: again replace a tags

File: src/components/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            <span>o.</span>
+          </a>
+        </div>
+
+        <div className="header__nav--search">
+          <input type="search" className="nav__search" placeholder="Search" />
+        </div>
+
+        <div className="header__nav--menu flex justify-between">
+          <a href="#" className="">
```
Comment (coldpigli) [inline src/components/Navbar/Navbar.jsx]: a tags here as well

**Review by coldpigli — APPROVED** [review]
Suggested minor changes

**Pull request merged**